src/antidote.app.src

```diff
@@ -15,6 +15,8 @@
                   antidote_pb,
                   erlzmq,
                   rand_compat,
+                  runtime_tools,
+                  tools,
```

Comment:
Should this be here or was this just for debugging?

Reply:
This was for debugging, but I don't see why they shouldn't be there in general.
The documentation says:
```[...]provides low footprint tracing/debugging tools suitable for inclusion in a production system.```